Wikipedia/Code/ExploreViewController.swift

```diff
@@ -28,7 +28,7 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
         navigationBar.shouldTransformUnderBarViewWithBar = true
         navigationBar.isShadowHidingEnabled = true
 
-        updateNotificationsCenterButton()
+        // updateNotificationsCenterButton()
```

Comment:
Here and elsewhere, did you want to keep these commented out for future or remove them?